src/pages/trainer/training/edit/components/toolbar/presentational.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import {ToolbarButton} from './buttons';
+import {
+  HeaderIcon, BoldIcon, ItalicIcon, CodeIcon, LinkIcon, ImageIcon,
+  BulletedListIcon, NumberedListIcon, QuoteIcon,
+} from './icons';
+
+interface IProps {
+  editor: HTMLTextAreaElement;
+  updateEditor: (editor: HTMLTextAreaElement, caret: string, offset: number) => void;
+}
+
+export const ToolbarComponent = (props: IProps) => {
```

Comment:
I think I named it presentation to avoid /toolbar/toolbar.tsx

But Ok, I agree to change it